src/main/common/maths.h

```diff
@@ -34,6 +34,8 @@
 
 // Use floating point M_PI instead explicitly.
 #define M_PIf       3.14159265358979323846f
+#define M_TWO_PIf   6.28318530717958647693f
+#define M_HALF_PIf  1.57079632679489661923f
```

Comment:
I'm pretty sure you don't need to do that as the compiler already replaces terms like `2 * M_PIf` with the resulting constant of `6.28318530717958647693f`.

So you can remove that and just use terms like `2 * M_PIf` and `M_PIf / 2` for better readability.

Reply:
(compiler is handling constant expressions, not preprocessor. But result is the same)

Reply:
I removed all this refactoring and will make a separate PR for it.
I only did it here because I noticed it.
We use sinApprox quite a lot, especially with the earth referencing of Angle mode and here in a Rescue, on multiple axes, so optimising this function for speed over readability is desirable.